llvm-bazel/llvm-project-overlay/llvm/BUILD

```diff
@@ -1730,6 +1730,7 @@ gentbl(
         srcs = ["lib/Target/" + target["name"] + "/TargetInfo/" + target["name"] + "TargetInfo.cpp"],
         hdrs = glob(["lib/Target/" + target["name"] + "/TargetInfo/*.h"]),
         copts = llvm_copts,
+        includes = ["lib/Target/" + target["name"]],
```

Comment:
Please add a TODO to remove this when the Bazel issue is fixed. Any idea what release that will be?